api/test/integration/test_agent_GET_endpoints.tavern.yaml

```diff
@@ -1898,6 +1898,42 @@ stages:
             rootcheck:
               sleep: !anyint
 
+  - name: Request-Wazuh-db-Internal
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/agents/000/config/wazuh-db/internal"
+      method: GET
+      headers:
+        Authorization: "Bearer {test_login_token}"
+    response:
+      status_code: 200
+      json:
+        error: !anyint
```

Comment:
Why can't we use specific error numbers here?

Reply:
I have changed from `!anyint` to error numbers in all the test cases.